config/systemConfig/ruby.yaml

```diff
@@ -0,0 +1,12 @@
+systemConfig:
+  - key: apiHttpLibraries
```

Comment:
are `faraday, rest-client` all these gem names? or module names?

For now, will recommend to add `.*` prefix to the regex so that we get maximum coverage, FP are ok for now

Reply:
Yes these are ruby gemnames.
For few module names I added (.*) as prefix in rule